tests/src/androidTest/java/com/squareup/duktape/DuktapeTest.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2015 Square, Inc.
+ * Copyright (C) 2016 Square, Inc.
```

Comment:
For what it’s worth, we typically set these at file-creation time and leave them as-is til the end of time.